src/System.Private.CoreLib/src/System/Type.CoreCLR.cs

```diff
@@ -83,7 +83,7 @@ public bool IsInterface
         //   param progID:     the progID of the class to retrieve
         //   returns:          the class object associated to the progID
         ////
-        public static Type GetTypeFromProgID(string progID, string? server, bool throwOnError)
+        public static Type? GetTypeFromProgID(string progID, string? server, bool throwOnError)
         {
             return RuntimeType.GetTypeFromProgIDImpl(progID, server, throwOnError);
```

Comment:
Should RuntimeType.GetTypeFromProgIDImpl also return Type? then?

Reply:
Good point. Such as `GetTypeFromCLSIDImpl`.

Reply:
Fixed.